jina/jaml/helper.py

```diff
@@ -33,6 +39,13 @@ def get_hashable_key(self, key):
         return key
 
     def construct_mapping(self, node, deep=True):
+        """
+        Build the mapping between
+
+        :param node: the node to traverse
+        :param deep:
```

Comment:
Can you add more info to this parameters?

Reply:
👍